bootstrap/mac.sh

```diff
@@ -1,2 +1,10 @@
 #!/bin/sh
+if hash brew 2>/dev/null; then
+    echo "Homebrew Installed"
+else
+    echo "Homebrew Not Installed\nDownloading..."
+    ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
```

Comment:
Oh wow, looks a bit scary + seems strange to download `master`, but this is what they suggest on http://brew.sh/.